README.md

```diff
@@ -101,6 +102,32 @@ you can export the `TranslateModule` to make sure you don't have to import it in
 export class SharedModule { }
 ```
 
+##### Standalone components
+
+For [`Standalone component`](https://angular.io/api/core/Component#standalone) based applications you have to include providers from `TranslateModule` inside your bootstrap configuration options
+
+```ts
+bootstrapApplication(AppComponent, {
+    providers: [
+        ...
+        importProvidersFrom(HttpClientModule), // or provideHttpClient() in Angular v15+
+        importProvidersFrom(TranslateModule.forRoot({
+            loader: {
+                provide: TranslateLoader,
+                useFactory: createTranslateLoader,
+                deps: [HttpClient]
+           }
+        }),
```

Comment:
missing `)` here:

`        })),`

Reply:
I'm sorry for the typo, just pushed a quick fix